agent/README.md

```diff
@@ -59,17 +59,23 @@ The following options are available:
   Teamscale uses this to map the coverage to the corresponding source code. Thus, this must be the exact code commit 
   from the VCS that was deployed. For an alternative see `teamscale-commit-manifest-jar`.
   
-  You can get the commit info from your VCS e.g. for Git via 
+  If **Git** is your VCS, you can get the commit info via
   
 ```bash
 echo `git rev-parse --abbrev-ref HEAD`:`git --no-pager log -n1 --format="%ct000"`
 ```
-
+  
   Note: Getting the branch does most likely not work when called in the build pipeline, because Jenkins, GitLab,
   Travis etc. checkout a specific commit by its SHA1, which leaves the repository in a detached head mode and thus 
   returns HEAD instead of the branch. In this case the environment variable provided by the build runner should be used 
   instead.
   
+  If **Subversion** is your VCS, you can get the timestamp part (but not the branch part) of the commit info via
+  
+  ```bash
+date -d `svn info --show-item last-changed-date` "+%s000"
```

Comment:
I think it would be good to also have the command on how to get the branch name from the subversion repository here.

Reply:
Depending on the version of GNU date, you may also use +"%s%3N" for exact milliseconds. Otherwise, we might risk uploading the coverage *before* the code commit.

Reply:
Also, you might want to add the `-r revision` parameter.

Reply:
> I think it would be good to also have the command on how to get the branch name from the subversion repository here.

That's tricky in SVN, as the branch name is just a convention. But it should be possible [to extract from the URL](https://stackoverflow.com/questions/5275924/what-svn-command-can-i-use-to-see-the-branch-im-currently-using). Will conduct some testing.